cloud/pkg/edgecontroller/manager/location.go

```diff
@@ -29,6 +29,16 @@ func (lc *LocationCache) PodConfigMapsAndSecrets(pod v1.Pod) (configMaps, secret
 		if v.Secret != nil {
 			secrets = append(secrets, v.Secret.SecretName)
 		}
+		if v.Projected != nil {
+			for _, source := range v.Projected.Sources {
+				switch {
+				case source.ConfigMap != nil:
+					configMaps = append(configMaps, source.ConfigMap.Name)
+				case source.Secret != nil:
+					secrets = append(secrets, source.Secret.Name)
```

Comment:
switch&case will only return one case, will there contain both source.ConfigMap.Name and source.Secret.Name?

Reply:
Yes, in projected volume in kubelet, it also use the switch case for sources. Ref:  https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/projected/projected.go#L254